docs/model-dev-guide/apis-howto/deepspeed/autotuning.rst

```diff
@@ -78,31 +84,25 @@ will describe specific use cases and the changes needed for each.
 DeepSpeedTrial
 ==============
 
-Determined's DeepSpeed Autotune works by inserting DeepSpeed configuration options into the
-``overwrite_deepspeed_args`` field of the ``hyperparameters`` dictionary which is seen by each
-trial. To take advantage of ``dsat``, you simply need to incorporate these "overwrite" values into
-your original configuration.
-
-.. note::
-
-   For more information about ``DeepSpeedTrial``, see :ref:`deepspeed-api`.
+In order to use Determined's DeepSpeed Autotune with ``DeepSpeedTrial`` there are two requirements.
```

Comment:
avoid references to specific counts. simplify "In order to" with "To".

To use Determine's DeepSpeed Autotune with ``DeepSpeedTrial``, you must meet the following requirements.